src/cmd/linuxkit/run_vcenter.go

```diff
@@ -199,9 +199,9 @@ func vCenterConnect(ctx context.Context, newVM vmConfig) (*govmomi.Client, *obje
 		log.Fatalln("Error locating default datacenter folder")
 	}
 
-	// Check if network connectivity is requested
+	// This code is shared between Push/Run, a network isn't needed for Pushing an image
 	var net object.NetworkReference
-	if *newVM.networkName != "" {
+	if (newVM.networkName != nil) && *newVM.networkName != "" {
```

Comment:
non-blocking nit: the parentheses are unnecessary

Reply:
ping @thebsdbox do you want to fix this?